docs/DEVELOPING_YOUR_FIRST_SKILL.rst

```diff
@@ -2,7 +2,7 @@
 Developing Your First Skill
 ============================
 
-The `Getting Started <GETTING_STARTED.rst>`_ guide showed how to set up and
```

Comment:
Some part of the documentation is hand-written (like getting started etc.). However, the other part is generated using the 'sphinx-apidoc' extension (like api , models reference). These files are basically populated during the build process (``make html``) and hence cannot be linked from rst. So, I shifted all links from ``rst`` to ``html`` for consistent linking. Since all the files (except samples) are hosted and referenced through RTD, this would work as expected. 

I tested this by having local doc generation and hosting the docs on RTD with a different project name.